File: URI/2896.py
N = int(input())

for i in range(N):
    N,k = list(map(int,input().split()))
    if N < k:
        print(N)
    elif N == k:
        print("1")
    else:
        extras = N // k
        print("{}".format(extras + N%k))

File: URI/2896.c
#include<stdio.h>

int main(){
    int N;
    scanf("%d",&N);

    for(int i=0; i<N; i++){
        int N,K;
        scanf("%d %d",&N,&K);

        if(N == K)
            printf("1\n");
        else if(N < K)
            printf("%d\n",N);
        else{
            int extras = N / K;
            printf("%d\n",extras + N%K);
        }
    }

    return 0;
}
